Release/QuickChat/docs/README.md

```diff
@@ -0,0 +1,29 @@
+# How to use the quickChat Module in your Zepeto Project ?
+
+The QuickChat module is the is used for sending in game predefined messages in a Zepeto Multiplayer world. 
+
+In this documentation, you will learn how to add a QuickChat module in your existing zepeto Project. If you are just getting started as a developer in Zepeto, we suggest you to  check out our [beginner guide](https://docs.zepeto.me/studio-item/lang-ko/docs) first. 
+
+
+## Adding the QuickChat Module into my project
+
+
+- Open the **Zepeto Module Importer** in the Zepeto project where you will add the QuickChat functionality.
+  
+- Import the QuickChat Module. while importing the module make sure that you also import the multiplay component as this functionality only works in a Multiplayer world. In case you already have a multiplay/ you are planning to add one on your own, feel free to uncheck the multiplay component.
+  
+- Drag and drop the **QuickChat Canvas** prefab located at "*Zepeto QuickChat Module > UI > Prefabs*" into your scene. 
```

Comment:
Do you think you can add screenshots for the step by step here?

Reply:
Done, sir :)